### PR TITLE
Fix/no jira catch malformed uri

### PR DESCRIPTION
**Pull request by petermason**
#### Description

HOT FIX -- resolve malformed uri headlines

### NO Issue
image

### MALFORMED
image

### CODE REVIEW

File: packages/article-update-header/index.tsx [hunk 1]
```diff
@@ -14,6 +14,15 @@ interface IProps {
 function ArticleUpdateHeader({ breaking = false, headline, updated }: IProps) {
   const dt = DateTime.fromISO(updated).setLocale("en-US");
   const { colours, fonts } = styleguide();
+
+  const getHeadlineSafely = () => {
+    try {
+      return decodeURIComponent(headline);
+    } catch (e) {
+      console.log("uri error: ", e);
```
[the remark this answers is not in the document]
Comment (petermason) [reply packages/article-update-header/index.tsx:22]: eg: 7% increase will fail

[the remark this answers is not in the document]
Comment (petermason) [reply packages/article-update-header/index.tsx:22]: we could return headline as a string?

[the remark this answers is not in the document]
Comment (newsuk-lee-daffen) [reply packages/article-update-header/index.tsx:22]: Sounds like a plan - good to have fallback if we can't decode the URI component

[the remark this answers is not in the document]
Comment (jonnyandrew) [reply packages/article-update-header/index.tsx:22]: As we've seen headline may be in an encoded format, wouldn't it be safer to fall back to an empty string?

[the remark this answers is not in the document]
Comment (newsuk-lee-daffen) [reply packages/article-update-header/index.tsx:22]: Okay, cool, let's go with that - correctly formatted or not at all :)

**Review by michalNews — APPROVED** [review]

File: packages/article-update-header/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (petermason) [reply packages/article-update-header/index.tsx:22]: reverting to empty string on error

**Review by michalNews — APPROVED** [review]

**Pull request merged**